CONTRIBUTING.md

```diff
@@ -47,137 +105,154 @@ the [Software Carpentry introduction to git][link_swc_intro].
 
-**5. Submit a [pull request][link_pullrequest].**
-
-A member of the development team will review your changes to confirm
-that they can be merged into the main code base.
-
-Pull requests titles should begin with a descriptive prefix:
-
-* ``ENH``: enhancements or new features ([example][enh_ex])
-* ``FIX``: bug fixes ([example][fix_ex])
-* ``TST``: new or updated tests ([example][tst_ex])
-* ``DOC``: new or updated documentation ([example][doc_ex])
-* ``STY``: style changes ([example][sty_ex])
-* ``REF``: refactoring existing code ([example][ref_ex])
-* ``CI``: updates to continous integration infrastructure ([example][ci_ex])
-* ``MAINT``: general maintenance ([example][maint_ex])
+## Making a change
 
-For example: `[ENH] Support for SB-reference in multi-band datasets`
+We appreciate all contributions to *fMRIPrep*,
+but those accepted fastest will follow a workflow similar to the following:
 
-For works-in-progress, add the ``WIP`` tag in addition to the descriptive prefix.
-Pull-requests tagged with ``[WIP]`` will not be merged until the tag is removed.
+1. **Comment on an existing issue or open a new issue referencing your addition.**<br />
+  This allows other members of the *fMRIPrep* development team to confirm that you aren't
+  overlapping with work that's currently underway and that everyone is on the same page
+  with the goal of the work you're going to carry out.<br />
+  [This blog][link_pushpullblog] is a nice explanation of why putting this work in up front
+  is so useful to everyone involved.
+  
+1. **[Fork][link_fork] the [fMRIPrep repository][link_fMRIPrep] to your profile.**<br />
+  This is now your own unique copy of *fMRIPrep*.
+  Changes here won't effect anyone else's work, so it's a safe space to explore edits to the code!
+  
+1. **[Clone][link_clone] your forked fMRIPrep repository to your machine/computer.**<br />
+  While you can edit files [directly on github][link_githubedit], sometimes the changes
+  you want to make will be complex and you will want to use a [text editor][link_texteditor]
+  that you have installed on your local machine/computer.
+  (One great text editor is [vscode][link_vscode]).<br />  
+  In order to work on the code locally, you must clone your forked repository.<br />  
+  To keep up with changes in the fMRIPrep repository,
+  add the ["upstream" fMRIPrep repository as a remote][link_addremote]
+  to your locally cloned repository.  
+    ```Shell
+    git remote add upstream https://github.com/poldracklab/fmriprep.git
+    ```
+    Make sure to [keep your fork up to date][link_updateupstreamwiki] with the upstream repository.<br />  
+    For example, to update your master branch on your local cloned repository:  
+      ```Shell
+      git fetch upstream
+      git checkout master
+      git merge upstream/master
+      ```
```

Comment:
Indentation keeps increasing...

Reply:
Yup, this is to make sure all these steps are rendered under item 1.